README Raspberry.md

```diff
@@ -42,15 +42,7 @@ sudo cp siridb-server /usr/local/bin/siridb-server
 
 Since version 2.0.35 it is possible to use the HTTP API to create and manage databases. However it is also possible to use the [admin tool](https://github.com/SiriDB/siridb-admin) for this.
```

Comment:
Misschien dit aanpassen in: 

Since 2.0.36 it is possible to use a build-in HTTP API for connecting with Grafana. The same API can also be used to create and manage databases. However, it is also possible to use ...siridb-http... and ...admin... . Both tools are required with older versions of SiriDB.